tiatoolbox/wsicore/wsireader.py

```diff
@@ -362,56 +365,57 @@ def _find_read_params_at_resolution(
         # Do we need sanity check for input form ?
         requested_location = np.array(location)
         requested_size = np.array(size)
-        lv0_to_read_lv_scale_factor = 1 / info.level_downsamples[read_level]
+        baseline_to_read_level_scale_factor = 1 / info.level_downsamples[read_level]
 
-        lv0_to_requested_lv_scale_factor = lv0_to_read_lv_scale_factor
-        lv0_to_requested_lv_scale_factor *= read_lv_to_requested_scale_factor
+        baseline_to_resolution_scale_factor = baseline_to_read_level_scale_factor
+        baseline_to_resolution_scale_factor *= read_level_to_resolution_scale_factor
```

Comment:
Probably not as issue but I would just be careful as `*=` can change the underlying value that is being referenced e.g. if it is a bumpy array. therefore it could be modifying `baseline_to_read_level_scale_factor` in addition to `baseline_to_resolution_scale_factor`. It might be goo to use a regular multiplication just to be safe.

Reply:
I agree.